contracts/dollar/hardhat.config.ts

```diff
@@ -28,9 +30,8 @@ const {
   API_KEY_COINMARKETCAP,
 } = process.env;
 
-const mnemonic = `${
-  MNEMONIC || "test test test test test test test test test test test junk"
-}`;
+const mnemonic = `${MNEMONIC || "test test test test test test test test test test test junk"
```

Comment:
We should delete the fallback, as I currently have this as part of the build process to render an `.env` file with this as the default value for `$MNEMONIC`